presto-hive/src/main/java/com/facebook/presto/hive/OrcFileWriterFactory.java

```diff
@@ -263,32 +270,16 @@ else if (com.facebook.hive.orc.OrcOutputFormat.class.getName().equals(storageFor
                 metadata.put(HOSTNAME_METADATA_KEY, HOSTNAME.get().get());
             }
 
+            OrcWriterOptions orcWriterOptions = buildOrcWriterOptions(session, schema);
+
             return Optional.of(new OrcFileWriter(
                     dataSink,
                     rollbackAction,
                     orcEncoding,
                     fileColumnNames,
                     fileColumnTypes,
                     compression,
-                    orcFileWriterConfig
```

Comment:
Extracted creation of the OrcWriterOptions to be able to assert it. Otherwise it will get buried in the writer.